events/elections/2018/README.md

```diff
@@ -0,0 +1,65 @@
+# 2018 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
+
+## Purpose
+
+The role of this election is to fill out the three(3) seats due for reelection this year on the [Kubernetes Steering Committee](https://github.com/kubernetes/steering).
+
+## Voting
+
+This election will  shape the future of Kubernetes as a project and community. You will be voting on three (3) seats up for election.
+Every elected community member will serve a two year term.
```

Comment:
I can't find any documentation on the two year term piece. The three seats were all one year terms. Not sure if we are missing docs on this or if I'm missing it somewhere.

Reply:
I couldn't find it codified either. However, if the terms were 1 year then the steering committee would turn over every year. The terms are currently staggered so that the committee never turns over all at once. 

This shouldn't be codified here though, it needs to be in the parent elections README, I'll work a PR for that.

Reply:
Right, elections README is perfect.

Reply:
elections README is perfect